examples/Serving models/Host GPT-2 (12-layer, 768-hidden, 12-heads, 117M parameters) .ipynb

```diff
@@ -0,0 +1,178 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h2>Setting Input</h2>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Encode a text inputs\n",
+    "text = \"Who was Jim Henson ? Jim Henson was a\"\n",
+    "indexed_tokens = tokenizer.encode(text)\n",
+    "\n",
+    "# Convert indexed tokens in a PyTorch tensor\n",
+    "tokens_tensor = th.tensor([indexed_tokens])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h2>Hosting GPT-2 Model</h2>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ionesio/workspace/venv/lib/python3.7/site-packages/syft-0.1.26a1-py3.7.egg/syft/frameworks/hook.py:342: TracerWarning: Converting a tensor to a Python float might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!\n",
+      "  response = method(*args, **kwargs)\n",
+      "/home/ionesio/workspace/venv/lib/python3.7/site-packages/syft-0.1.26a1-py3.7.egg/syft/frameworks/hook.py:342: TracerWarning: Converting a tensor to a Python index might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!\n",
+      "  response = method(*args, **kwargs)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Grid nodes publish datasets online and are for EXPERIMENTAL use only.Deploy nodes at your own risk. Do not use OpenGrid with any data/models you wish to keep private.\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'success': True}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "traced_model = th.jit.trace(model, (tokens_tensor,))\n",
+    "\n",
+    "# Grid Node\n",
+    "bob = gr.WebsocketGridClient(hook, \"http://localhost:3000/\", id=\"Bob\")\n",
+    "bob.connect()\n",
+    "\n",
+    "# Host GPT-2 on Bob worker\n",
+    "bob.serve_model(traced_model, model_id=\"GPT-2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h2>Running Inference</h2>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Predicted text:   Who was Jim Henson? Jim Henson was aage\n",
+      "CPU times: user 26.5 ms, sys: 0 ns, total: 26.5 ms\n",
+      "Wall time: 174 ms\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
We should try something like this: https://github.com/huggingface/pytorch-transformers/blob/master/examples/run_generation.py#L106 :eyes:

Reply:
Done! (d491e0a)